modules/Mobile/actions/getScrollContent.php

```diff
@@ -62,8 +62,16 @@ public function process(crmtogo_API_Request $request) {
 			}
 			$id = $ws_entity2.'x'.$adb->query_result($records, $i, $entityidfield);
 
-			$output .= '<li><a class="ui-btn ui-btn-icon-right ui-icon-carat-r" target="_self" href="?_operation=fetchRecord&record='.$id.'" target="_self">'
-				.$firstname.'&nbsp;'.$lastname.'</a></li>';
+			$fetchRecordUrl = '/modules/Mobile/index.php?_operation=fetchRecord&record=' . $id;
```

Comment:
Are you sure that the absolute path is correct in all cases? Can we leave it the way it was, just the query parameters?

`$fetchRecordUrl = '?_operation=fetchRecord&record=' . $id;`

instead of

`$fetchRecordUrl = '/modules/Mobile/index.php?_operation=fetchRecord&record=' . $id;`

just to make it as similar to the way it was as possible.

Reply:
ok I will change it